src/gsi/read_radar.f90

```diff
@@ -2154,18 +2176,14 @@ subroutine read_radar(nread,ndata,nodata,infile,lunout,obstype,twind,sis,hgtl_fu
   ibadstaheight=0
   notgood=0
   notgood0=0
-  nread=0
```

Comment:
Comment for deter_sfc_mod.f90:  It will give the same result because for both the value for nfldsfc is multiplied by one.

I have made the other changes.  It reduces the possibility of a small round-off error.

Comment for read_radar.f90: It is set to 0 in the read_obs routine (one level up).  However I will add it back in.  Doesn't really make any difference.